### DISCUSSION TITLE
ioBroker MQTT: need single variables for each setting

### DISCUSSION THREAD

**Discussion by rug1024** [revk/xESP32-Faikout#469] [General]
In ioBroker MQTT the readings from Faikin are imported as a single line of parameters, thus making it for me as a "noob" hard to process for further use in the home automatization:

![grafik](https://github.com/user-attachments/assets/be11a90b-388b-46d4-a3aa-db89bd04ed18)

As a comparision, here a view of how the settings of my solar inverter are imported via MQTT:

![grafik](https://github.com/user-attachments/assets/539e6c7c-1c10-4e0e-8318-dddc5f1b0932)
Every single position is one data point, these can easily adressed from within scripts, marked for recording in the database, etc.

Is it possible for Faikin to export MQTT in the same way? I tried turning on Home Assistant support, but the format used there seems very different.

**Comment by revk** [comment]
There is a setting, I think fixstatus, which changes from min/ave/max array to a single value.

**Reply by rug1024** [reply]
No, I tried that. Does not help, there is still a whole string of values.

![grafik](https://github.com/user-attachments/assets/0f69484b-3966-406e-b499-5151d00e9b20)


The solar inverter software I mentioned above... it's here: https://github.com/tbnobody/OpenDTU
Maybe you can take a look how the programmer did it there?

**Reply by revk** [reply]
Wait, you want separate messages for each value? That would be crazy, surely what you have can cope?
